awsconfigs/common/dex/patches/disable-nodeport.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: dex
+spec:
+  ports:
+  - name: dex
+    port: 5556
+    protocol: TCP
+    targetPort: 5556
+    nodePort: null
```

Comment:
remove this line?

Reply:
`nodePort: 32000` will still be showing if removed